nesta/production/schemas/tier_1/schema_transformations/eu_github.json

```diff
@@ -0,0 +1,19 @@
+[

```

Comment:
This file looks like a duplicate of github_projects_and_users - is that true?

Reply:
They are almost the same but the field names of the original datasets are different.

Reply:
I named the .json files in a similar fashion to the datasets given to me.